index.css

```diff
@@ -1,101 +1,101 @@
-/* This is to use a unified border size */
```

Comment:
I do not think that this file needs to be removed or modified unless there is a hotfix.

Reply:
it's a comment though, what differences does it make let's leave it back.